operations/commit_queue.go

```diff
@@ -354,12 +345,14 @@ func (p *mergeParams) uploadMergePatch(conf *ClientSettings, ac *legacyClient) e
 		return errors.Wrap(err, "can't generate patches")
 	}
 
-	if p.message == "" && commitCount != 0 {
-		message, err := gitCommitMessages(ref.Branch, p.ref)
+	if commitCount > 0 {
+		commitMessage, err := gitCommitMessages(ref.Branch, p.ref)
 		if err != nil {
-			return errors.Wrap(err, "can't get commit messages")
+			errors.Wrap(err, "can't get commit messages")
 		}
-		patchParams.Description = message
+		patchParams.Description = fmt.Sprintf("Commit Queue Merge: %s", fmt.Sprintf(commitFmtString, commitMessage, ref.Owner, ref.Repo, ref.Branch))
```

Comment:
I think having two fmt.Sprintfs on one line like this is awkward to read--could you break the second one onto its own line? and rather than have a one-line else you can maybe set patchParams.Description to the no commit default before the if statement.

Reply:
done